src/refiners/conversion/utils.py

```diff
@@ -135,6 +135,7 @@ def local_hash(self) -> str:
 
     def check_local_hash(self) -> bool:
         """Check if the sha256 hash of the file in the local hub is correct."""
+        assert self.expected_sha256 is not None, f"{self.repo_id}/{self.filename} has no expected sha256 hash"
         if self.expected_sha256 != self.local_hash:
```

Comment:
nit: I think `local_hash` could be renamed into `local_sha256_hash` (to echo `hf_sha256_hash`)